### PR TITLE
feat: added the images component

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Image component.
- Responsive Images - circular and squared

image

image

- Squared Images - small, medium, and large

image

- Circular Images - small, medium, and large

image

### CODE REVIEW

File: stylesheets/components/images/images.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+.round-img img{
+    border-radius: 50%;
+    width: 300px;
+    height: 300px;
+}
+
+.squared-img img{
+    width: 300px;
+    height: 300px;
```
Comment (anshulraheja) [inline stylesheets/components/images/images.css:18]: Try to use rem unit instead of pixel. Rest is fine and you can merge!

**Review by anshulraheja — APPROVED** [review]

File: docs/images.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <div class="component-wrapper squared-images-wrapper">
+                <h1 class="wrapper-heading">Squared Images</h1>
+                <hr>
+                <div class="description wrapper-description">
+                    Images can be square-shaped using class "sq-img" in your HTML code. Use the classes "small",
+                    "medium" and "large" for the preferred sizes.
+                </div>
+                <div class="component-showcase sq-img-showcase">
+                    <img src="https://images.unsplash.com/photo-1637152630314-9e73782b9cb9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
+                        alt="" class="sq-img small">
```
Comment (dhimanaishwarya) [inline docs/images.html:73]: you can add alt for img

File: stylesheets/colors.css [hunk 3]
```diff
@@ -5,37 +5,35 @@
...
-
     --simple-alerts-section: #f3eded;
     --alerts-with-icons-section: #ebe7df;
 
-
     --badges-on-icons-section: #f0e7dc;
     --badges-on-avatars-section: #f3d6d4;
     --badge-default-bg: #af2b21;
     --badge-default-text: white;
     --badge-border-default: white;
```
Comment (deekshithmd) [inline stylesheets/colors.css]: using color code will be good

**Comment by SinchanaMS** [comment]
Made the PR changes, merging to dev.

**Pull request merged**